app/src/main/res/values/styles.xml

```diff
@@ -14,6 +14,34 @@
         <item name="android:textColor">@android:color/white</item>
     </style>
 
+    <!-- Facebook Button Style-->
+    <style name="ThemeOverlay.FacebookButton" parent="ThemeOverlay.AppCompat.Dark">
+        <item name="colorButtonNormal">@color/facebook_color</item>
+        <item name="android:textColor">@android:color/white</item>
```

Comment:
Make a style that inherits from ThemeOverlay.AppCompat.Dark and have the android:textColor item only. Then make that the all this styles that use that item inherits from this new style.